tests/test-extensions/metadata-extension/src/main/java/org/mule/test/metadata/extension/MetadataOperations.java

```diff
@@ -163,6 +167,13 @@ public boolean typeWithDeclaredSubtypesMetadata(Shape plainShape, Rectangle rect
     return false;
   }
 
+  @Query(translator = MetadataExtensionQueryTranslator.class,
+      entityResolver = MetadataExtensionEntityResolver.class,
+      outputResolver = NativeQueryOutputResolver.class)
```

Comment:
I get The point, but when you are using only native query, de take it as a common case of output resolver... still, i'd say +1 to chaging it